src/server/config/config.js

```diff
@@ -61,13 +61,25 @@ export class Config {
     if (_.isPlainObject(key)) {
       config = override(config, key);
     } else {
-      _.set(config, key, value);
+      _.set(config, key, this._replaceEnvVars(value));
     }
 
     // attempt to validate the config value
     this._commit(config);
   }
 
+  _replaceEnvVars(value) {
```

Comment:
Can we add tests for all of these cases in addition to the env variable not being specified?

Reply:
Hi @tylersmalley 

- OK, I will add a test when env variable is not specified.
- Then, for additional cases, currently, `_.isString(value)` and `_.isArray(value)` are covered. Only `_.isPlainObject(value)` is not covered. I will change the test to cover it.